libnmstate/nm/ipv4.py

```diff
@@ -20,8 +20,17 @@
 from . import nmclient
 
 
-def create_setting(config):
-    setting_ipv4 = nmclient.NM.SettingIP4Config.new()
+def create_setting(config, base_remote_conn):
```

Comment:
Please do change `base_remote_conn` to `base_con_profile` to have it uniform in nmstate (also  for IPv6).

Reply:
Done. Thanks.